.buildkite/commands/configure-for-release.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash -eu
 
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
+  echo "This script must be 'source'd (instead of being called directly as an executable) to work properly"
+  exit 1
+fi
```

Comment:
When a script is meant to be sourced, the shebang is ignored. (I learned that the hard way, but it's mentioned [in Wikipedia](https://en.wikipedia.org/wiki/Shebang_(Unix)#cite_note-9:~:text=The%20shebang%20line%20is%20usually%20ignored%20by%20the%20interpreter%2C%20because%20the%20%22%23%22%20character%20is%20a%20comment%20marker%20in%20many%20scripting%20languages%3B%20some%20language%20interpreters%20that%20do%20not%20use%20the%20hash%20mark%20to%20begin%20comments%20still%20may%20ignore%20the%20shebang%20line%20in%20recognition%20of%20its%20purpose.%5B9%5D) and there are StackOverflow questions like [this one](https://stackoverflow.com/questions/49080559/source-script-does-not-honor-bin-bash-ex-and-exit-when-a-command-fails) about it.)

That's all to say, we should replicate the flags manually after ensuring the script is sourced.

```suggestion
if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
  echo "This script must be 'source'd (instead of being called directly as an executable) to work properly"
  exit 1
fi

set -e
set -u
```

Or... we might remove them entirely and rely on the caller to manage that? 🤔 ...Nah, I think being extra cautious here is best.